adhocracy-plus/assets/scss/_utility.scss

```diff
@@ -140,3 +140,13 @@
 .u-border {
     border: 1px solid $border-color;
 }
+
+.u-success {
+    color: $brand-success;
+}
+
+.Oval-Copy-8 {
```

Comment:
what does this do? I think it should go into the components and named after the component. In utilities, the classes should only do one thing and be named like that